docs/website/docs/plus/features/projects.md

```diff
@@ -0,0 +1,148 @@
+```text
+.
+├── dlt_example_project
+│    ├── .dlt/                 # folder containg dlt configrations and profile settings
+│    │   ├── config.toml
+│    │   ├── dev.secrets.toml  # credentials for access profile 'dev'
+│    │   └── secrets.toml      
+│    ├── sources/              # modules containing the source code for sources 
+│    │   └── github.py         # source code for a github source  
+│    ├── destinations/         # modules containing the source code for destinations  
+│    ├── transformations/      # modules containing the source code for transformations 
+│    ├── .gitignore
+│    └── dlt.yml               # the main dlt manifest file
+└── pyproject.toml             # the python manifest file for the package
+```
+
+### The dlt manifest file (dlt.yml)
+
+The main component of a dlt+ Project is the dlt manifest file (`dlt.yml`). It marks the root of your project and contains the main configurations. Here you can declare all of your data platform entities in a yaml format. It contains the following sections:
+
+#### Sources
+
+This section lets you define sources either declaratively or by referencing an implementation from a python module inside `sources/`. In the example below, two sources are declared:  
+1. a dlt REST API source whose parameters are passed within the manifest
+2. a github source whose source code inside `sources/github.py` is referenced
+  
+```yaml
+sources:
+  pokemon:
+    type: dlt.sources.rest_api.rest_api
+    client:
+      base_url: https://pokeapi.co/api/v2/
+    resource_defaults:
+      endpoint:
+        params:
+          limit: 1000
+    resources:
+      - pokemon
+      - berry
+
+  github:
+    type: dlt_example_project.github.source
+```
+#### Destinations  
+  
+The destinations section defines dlt destinations in a similar way to how you would define them in a pure Python dlt project. As with sources, you can also reference custom implementations of destinations from inside the `destinations/` folder.
+  
+```yaml  
+destinations:
+    duckdb:
+        type: duckdb
+```
+
+#### Pipelines 
+
+Pipelines can be used to load data from sources to destinations. The pipeline defined below loads data from the github source to a dataset named "github_events_dataset" inside the duckdb destination.
+
+```yaml
+  github_pipeline:
+    source: github
+    destination: duckdb
+    dataset_name: github_events_dataset
+```
+You can declare all arguments of `dlt.pipeline` in this section. For a full list of arguments, refer to the [docstrings](https://github.com/dlt-hub/dlt/blob/71b4975c70d1931750b3245e919a520a2400e870/dlt/pipeline/__init__.py#L30).  
+  
+
+#### Datasets
+
+The datasets section defines datasets that live on a destination (defined in the destinations section). Read more about datasets in dlt+ [here](../core-concepts/datasets.md).  
+  
+```yaml
+datasets:
+    github_events_dataset:
+        "on":
+            - duckdb
+```
+
+#### Caches  
```

Comment:
caches and transformations are experimental at this point

Reply:
We can mark it as experimental 🧪